packages/lesswrong/components/events/EventsHome.tsx

```diff
@@ -0,0 +1,355 @@
+      if (!highlightedEvent && !result.globalEvent) {
+        highlightedEvent = result
+      }
+    })
+    // otherwise, just show the first event in the list
+    if (!highlightedEvent) highlightedEvent = results[0]
+  }
+  
+  let loadMoreButton = showLoadMore && <button className={classes.loadMore} onClick={() => loadMore(null)}>
+    Load More
+  </button>
+  if (loading && results?.length) {
+    loadMoreButton = <CircularProgress size={16} />
+  }
+
+  return (
+    <>
+      <AnalyticsContext pageContext="EventsHome">
+        <div>
+          <HighlightedEventCard event={highlightedEvent} loading={loading} />
+        </div>
+
+        <div className={classes.section}>
+          <div className={classes.sectionHeadingRow}>
+            <h1 className={classes.sectionHeading}>Events</h1>
+            <div className={classes.sectionDescription}>
+              Connect with people near you and around the world who are trying to find the best ways to help others.
+              Learn, discuss, or just hang out with like-minded people.
+            </div>
+          </div>
+
+          <div className={classes.eventCards}>
+            <div className={classes.filters}>
+              <div className={classes.where}>
+                Showing events near {mapsLoaded
+                  && <div className={classes.geoSuggest}>
+                      <Geosuggest
+                        placeholder="Location"
+                        onSuggestSelect={(suggestion) => {
+                          if (suggestion?.location) {
+                            saveUserLocation({
+                              ...suggestion.location,
+                              known: true,
+                              gmaps: suggestion.gmaps
+                            })
+                          }
+                        }}
+                        initialValue={queryLocation?.label}
+                      />
+                    </div>
+                }
+              </div>
+              <Select
+                className={classes.filter}
+                value={modeFilter}
+                onChange={(e) => setModeFilter(e.target.value)}>
+                  <MenuItem key="all" value="all">In-person and online</MenuItem>
+                  <MenuItem key="in-person" value="in-person">In-person only</MenuItem>
+                  <MenuItem key="online" value="online">Online only</MenuItem>
+              </Select>
+            </div>
+
+            <EventCards events={results} loading={loading} numDefaultCards={6} />
+            
+            <div className={classes.loadMoreRow}>
+              <Button variant="text" color="primary" onClick={openEventNotificationsForm} className={classes.eventNotificationsBtn}>
+                <NotificationsNoneIcon className={classes.eventNotificationsIcon} />
+                {currentUser?.nearbyEventsNotifications ? `Edit my notification settings` : `Sign up for notifications`}
+              </Button>
```

Comment:
This is copied from the current Community page.

I think we should probably move this button above the events list, but I'm not sure where yet. I'd like to release this page this week so I'll probably move it up in a future PR.